build.sbt

```diff
@@ -12,6 +12,7 @@ lazy val scalaJavaTimeVersion        = "2.0.0"
 inThisBuild(Seq(
   homepage := Some(url("https://github.com/gemini-hlsw/gsp-math")),
   addCompilerPlugin("org.typelevel" %% "kind-projector" % kindProjectorVersion),
+  Global / onChangedBuildSource := ReloadOnSourceChanges
```

Comment:
Will this be a good default to set on sbt-gsp?

Reply:
Yeah, if you think it's good then let's do it. I can't think of a situation when I *don't* want to reload immediately.